draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -176,32 +176,39 @@
    
    <t>
            This document is a second edition of the CDNI Control Interface / Triggers, 
-           which defines a new version, "v2", of the interface objects. The new objects replace 
-           the trigger interface's main objects, the "ci-trigger-command" object and its matching
-           "ci-trigger-status" object to support <xref target="REST">RESTful</xref> semantics in a way that improves the
-           interface's flexibility, extensibility and interoperability, and allows encoding the interface using OpenAPI.
-           The document also provides a Trigger Extension mechanism that MAY be used to provide further instruction
-           on the trigger execution.
-           Additionally, this second edition includes cascaded CDN error propagation for improved 
-           trigger execution monitoring.
+	   which defines a new version, "v2", of the interface objects. 
+	   The new objects replace the trigger interface's main objects, the "ci-trigger-command" object and its matching
+	   "ci-trigger-status" object with "ci-trigger.v2" object to support <xref target="REST">REST</xref> architectural 
+	   style in a way that improves the interface's flexibility, extensibility and interoperability, and allows encoding 
+	   the interface using OpenAPI.  
+   </t>
+   <t>
+	   The second edition of the CI/T interface further allows use of separate Control interface endpoints for 
+	   content and metadata.
+   </t>
+   <t>
+	   The document also provides a Trigger Extension mechanism that MAY be used to provide further instruction
+	   on the trigger execution.
+   </t>
+   <t>
+	   This second edition also includes cascaded CDN error propagation and extended trigger status reporting 
+	   for improved trigger execution monitoring, as well as use of external object lists for improved scale and 
+	   integration of trigger-based APIs with existing content workflows.
    </t>
 
+
    <t><list style="symbols"><t><xref target="model-for-cdni-triggers"/> outlines the model for the CI/T interface at a high
       level.</t>
       
-   <t><xref target="trigger-specification"/> describes the trigger attributes used for specifying the trigger's targets and for controlling the trigger's execution.</t>
-
-   <t><xref target="collections-of-trigger-status-resources"/> describes collections of Trigger Status Resources.</t>
 
    <t><xref target="cdni-trigger-interface"/> defines the web service provided by the downstream CDN.</t>
 
-   <t><xref target="cit-objects-properties-and-encoding"/> lists properties of CI/T Commands and Status Resources.</t>
-
-   <t><xref target="trigger-spec-objects"/> describes the Trigger's Spec object.</t>
+   <t><xref target="cit-objects-properties-and-encoding"/> lists properties of CI/T Resources.</t>
 
    <t><xref target="trigger-extension-objects"/> describes the Trigger's Extensions object.</t>
```

Comment:
In the rendering, it appears the other sections are top level sections, but this one is an exception, diving deep into the implementation of Trigger Extensions.  Might it make sense to include this with the previous item, and maybe phrase it "Section 4 lists the properties of CI/T Resources, including the optional Trigger Extensions"?

Reply:
this is a leftover, shouldn't be there 
i did call out the trigger extensions as well as trigger specs in the corrected text
<t><xref target="cit-objects-properties-and-encoding"/> defines encoding of the standard CI/T objects, and introduces trigger spec and trigger extension types.</t>